scanpy/datasets/_datasets.py

```diff
@@ -298,38 +398,70 @@ def pbmc3k() -> ad.AnnData:
     Returns
     -------
     Annotated data matrix.
+
+    Examples
+    --------
+    >>> import scanpy as sc
+    >>> sc.datasets.pbmc3k()
+    AnnData object with n_obs × n_vars = 2700 × 32738
+        var: 'gene_ids'
     """
     url = "https://falexwolf.de/data/pbmc3k_raw.h5ad"
     adata = read(settings.datasetdir / "pbmc3k_raw.h5ad", backup_url=url)
     return adata
 
 
+@doctest_internet
 @filter_oldformatwarning
 @check_datasetdir_exists
-def pbmc3k_processed() -> ad.AnnData:
-    """Processed 3k PBMCs from 10x Genomics.
+def pbmc3k_processed() -> AnnData:
+    """\
+    Processed 3k PBMCs from 10x Genomics.
 
     Processed using the basic tutorial :doc:`/tutorials/basics/clustering-2017`.
 
+    For preprocessing, cells are filtered out that have few gene counts or too high a `percent_mito`.
+    The counts are logarithmized and only genes marked by :func:`~scanpy.pp.highly_variable_genes` are retained.
+    The :attr:`~anndata.AnnData.obs` variables `n_counts` and `percent_mito` are corrected for
+    using :func:`~scanpy.pp.regress_out`, and values are :func:`scanpy.pp.scale`\\ d and clipped.
```

Comment:
The `scale`d formatting looks weird on docs because it comes out like this IMO
<img width="180" alt="Screenshot 2024-06-04 at 14 39 56" src="https://github.com/scverse/scanpy/assets/43999641/ad014b1b-0724-43c0-a87a-c24d81736b78">